db/migrate/201604131212938_create_exemptions.rb

```diff
@@ -0,0 +1,34 @@
+class CreateExemptions < ActiveRecord::Migration
+
+  def change
+
+    create_table :flood_risk_engine_exemptions do |t|
+      t.string  :code
+      t.string  :summary
+      t.text    :description
+      t.string  :url
+      t.integer :category
+
```

Comment:
@autotelik Is it worth leaving out url category valid_from and valid_to until they are needed?